.pipelines/stages/jobs/py-packaging-job.yml

```diff
@@ -81,6 +81,12 @@ jobs:
   - name: os
     value: ${{ parameters.os }}
 
+  - name: feed_name
+    ${{ if eq(parameters.cuda_version, '12.2') }}:
```

Comment:
Better also check `eq(parameters.ep, 'cuda`)

Reply:
why

Reply:
All other packages (DirectML, ROCm) needs the original feed.

Reply:
Make sense. In the future, when cuda default value switch to 12.2. We do not need change here.